1.1/translations/geo_de.ttl

```diff
@@ -0,0 +1,288 @@
+# baseURI: http://www.opengis.net/ont/geosparql
+
+@base <http://www.opengis.net/ont/geosparql> .
+
+@prefix : <http://www.opengis.net/ont/geosparql#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix spec11: <http://www.opengis.net/spec/geosparql/1.1/specification.html#> . # TODO: use PID IRI
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:gmlLiteral
+	skos:prefLabel "GML Literal"@de ;
+	skos:definition """Eine GML Serialisierung eines geometrischen Objekts."""@de .
+
+:gmlLiteral
+	skos:prefLabel "Well-known Text Literal"@de ;
+	skos:definition """Eine Well-Known Text Serialisierung eines geometrischen Objekts."""@de .
+	
+:geoJSONLiteral
+	skos:prefLabel "GeoJSON Literal"@de ;
+	skos:definition """Eine GeoJSON Serialisierung eines geometrischen Objekts."""@de .
+	
+:kmlLiteral
+	skos:prefLabel "KML Literal"@de ;
+	skos:definition """Eine KML Serialisierung eines geometrischen Objekts."""@de .
+	
+:dggsLiteral
+	skos:prefLabel "DGGS Literal"@de ;
+	skos:definition """Eine Textserialisierung eines Discrete Global Grid (DGGS) geometrischen Objekts."""@de ;
```

Comment:
I think you need "Eine Textserialisierung eines Discrete Global Grid **System** (DGGS) geometrischen Objekts." so that the acronym is expanded correctly.